src/snowflake/connector/bind_upload_agent.py

```diff
@@ -19,17 +25,13 @@
 
 
 class BindUploadAgent:
-    _STAGE_NAME = "SYSTEMBIND"
-    _CREATE_STAGE_STMT = (
-        f"create or replace temporary stage {_STAGE_NAME} "
-        "file_format=(type=csv field_optionally_enclosed_by='\"')"
-    )
 
     def __init__(
         self,
         cursor: SnowflakeCursor,
         rows: list[bytes],
         stream_buffer_size: int = 1024 * 1024 * 10,
+        use_scoped_temp_object: bool = True,
```

Comment:
do we need this as part of the public api/do we want users to control the behavior?

Reply:
Will change this to private